### ISSUE TITLE
CLI Tool is incorrectly handling multiple machines inside one file.

### ISSUE THREAD

**Issue by SimeonC** [statelyai/xstate-tools#107]
Using `@xstate/cli@0.1.4`, if you have 2 `createMachine` calls in the same file they both end up pointing to the same tsTypes import.

Before CLI run.

```ts
const m1 = createMachine({
  tsTypes: {}
});

const m2 = createMachine({
  tsTypes: {}
});
```

After CLI Run

```ts
const m1 = createMachine({
  tsTypes: {} as import("./machineTemp.typegen").Typegen0
});

const m2 = createMachine({
  tsTypes: {} as import("./machineTemp.typegen").Typegen0 // NOTE: This should be Typegen1
});
```
Generated Typegen file;
```ts
// This file was automatically generated. Edits will be overwritten

export interface Typegen0 {
  '@@xstate/typegen': true;
  eventsCausingActions: {};
  internalEvents: {
    'xstate.init': { type: 'xstate.init' };
  };
  invokeSrcNameMap: {};
  missingImplementations: {
    actions: never;
    services: never;
    guards: never;
    delays: never;
  };
  eventsCausingServices: {};
  eventsCausingGuards: {};
  eventsCausingDelays: {};
  matchesStates: undefined;
  tags: never;
}
export interface Typegen1 {
  '@@xstate/typegen': true;
  eventsCausingActions: {};
  internalEvents: {
    'xstate.init': { type: 'xstate.init' };
  };
  invokeSrcNameMap: {};
  missingImplementations: {
    actions: never;
    services: never;
    guards: never;
    delays: never;
  };
  eventsCausingServices: {};
  eventsCausingGuards: {};
  eventsCausingDelays: {};
  matchesStates: undefined;
  tags: never;
}
```

**Comment by mattpocock** [comment]
Fixed in #112

**Comment by mattpocock** [comment]
Deployed to prod
